### PR TITLE
feat:visibility isolate

### PR DESCRIPTION
**Pull request by wildeslam**

### CODE REVIEW

File: arex-storage-model/src/main/java/com/arextest/model/replay/AppVisibilityLevelEnum.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.arextest.model.replay;
+
+import lombok.AllArgsConstructor;
+
+/**
+ * @author wildeslam.
+ * @create 2023/11/1 16:55
+ */
+@AllArgsConstructor
+public enum AppVisibilityLevelEnum {
```
Comment (coryhh) [inline arex-storage-model/src/main/java/com/arextest/model/replay/AppVisibilityLevelEnum.java:10]: the enum class not use

File: arex-storage-config/src/main/java/com/arextest/config/model/vo/UpdateApplicationRequest.java [hunk 2]
```diff
@@ -15,4 +16,8 @@ public class UpdateApplicationRequest {
   private String appId;
   private String appName;
   private Set<String> owners;
+  /**
+   * @see com.arextest.model.replay.AppVisibilityLevelEnum .
+   */
+  private Integer visibilityLevel;
```
Comment (coryhh) [inline arex-storage-config/src/main/java/com/arextest/config/model/vo/UpdateApplicationRequest.java:22]: the missing verification of the parameter

**Comment by coryhh** [comment]
suggestion: the storage service participates in the front-end page interaction. Is it better to move to arex-api?

File: arex-storage-config/src/main/java/com/arextest/config/model/vo/UpdateApplicationRequest.java (hunk 2 above)
Comment (binyu1005) [reply arex-storage-config/src/main/java/com/arextest/config/model/vo/UpdateApplicationRequest.java:22]: Replace Integer with int to maintain consistency.

**Review by QizhengMo — APPROVED** [review]

**Review by coryhh — APPROVED** [review]

**Pull request merged**